Gallery/Topography/NCL_topo_4.py

```diff
@@ -0,0 +1,119 @@
+   - Masking ocean elevation data
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/topo_4.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/topo_4_lg.png
+
+Note:
+    In the original NCL script, the ETOPO2 dataset was used. For this example,
+    we use the newer 1' data, ETOPO1.
+"""
+
+###############################################################################
+# Import packages:
+
+import matplotlib.pyplot as plt
+import xarray as xr
+import numpy as np
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import cmaps
+
+import geocat.viz as gv
+import geocat.datafiles as gdf
+
+###############################################################################
+# Read in data:
+
+# Note: The dataset used in this example is a subset of the ETOPO1 global elevation dataset which can be downloaded here: https://www.ngdc.noaa.gov/mgg/global/
+
+# Open a netCDF file using xarray
+ds = xr.open_dataset(gdf.get('netcdf_files/aus_elev.nc'))
+
+# Select elevation data
+ds = ds.z
+
+###############################################################################
+# Plot
+
+# Generate figure and set size
+plt.figure(figsize=(10, 10))
+
+# Generate axes, using Cartopy
+projection = ccrs.PlateCarree()
+ax = plt.axes(projection=projection)
+
+# Add coastlines
+ax.coastlines(zorder=10)
+
+# Add state/territory borders
+states_provinces = cfeature.NaturalEarthFeature(
+    category='cultural',
+    name='admin_1_states_provinces_lines',
+    scale='50m',
+    facecolor='none')
+ax.add_feature(states_provinces, zorder=5, linewidth=0.4)
+
+# Select NCL colormap and truncate to remove blue from lower end
+cmap = cmaps.OceanLakeLandSnow
+newcmap = gv.truncate_colormap(cmap=cmap, minval=0.01, maxval=1)
+
+# Plot the elevation data
+elev = ds.plot.imshow(ax=ax,
+                      transform=projection,
+                      cmap=newcmap,
+                      vmin=0,
+                      vmax=4000,
+                      add_colorbar=False)
+
+# Set extent of the plot
+ax.set_extent([110, 155, -45, -5])
+
+# Add ocean mask
+ax.add_feature(cfeature.OCEAN, zorder=2)
+
+# Add colorbar
+cbar = plt.colorbar(ax=ax,
+                    mappable=elev,
+                    orientation='horizontal',
+                    pad=0.1,
+                    shrink=0.85,
+                    ticks=np.arange(0, 4500, 500))
+cbar.ax.tick_params(
+    size=0,
+    labelsize=14)  # Remove the tick marks from the colorbar, set label size
+cbar.ax.xaxis.set_tick_params(pad=10)
+
+# Use geocat-viz utility function to add left and right titles
+gv.set_titles_and_labels(ax,
+                         lefttitle='elevation',
+                         righttitle='m',
+                         maintitle='ETOPO1',
+                         maintitlefontsize=23)
+
+# Remove default x and y labels
+plt.xlabel("")
+plt.ylabel("")
```

Comment:
```suggestion
gv.set_titles_and_labels(ax,
                         lefttitle='elevation',
                         righttitle='m',
                         maintitle='ETOPO1',
                         maintitlefontsize=23,
                         xlabel="",
                         ylabel="")
```
You can use `gv.set_title_and_labels` to remove the labels rather than making two other calls explicitly